.travis.yml

```diff
@@ -8,8 +8,9 @@ python:
   - "3.5"
   - "3.6"
   - "3.7"
+  - "3.8"
 
 install:
-  - pip install tox-travis
+  - pip install tox-travis cython
```

Comment:
Why? Shouldn't pyproject.toml take care of it?

Reply:
Experimentally it did not with the version of pip I was testing with locally.  I don't think that the `pyproject.toml` work is settled enough to rely on widely yet.

Reply:
This was the reason that I kept the C sources... so that legacy applications which might depend on old pip / setuptools might have a chance to use datrie. I guess you want an explicit `ImportError` of Cython instead, then?

Reply:
If you *really want* to go this way I would request you to bump the version to `0.9.0` - because it is a breaking change not a bug fix (think, semver).

Reply:
It is also possible to run cython as part of the release process and still include the c-files in the sdist, however that just seems like a trap for user.

The better solution is to provide as many wheels as we can so that most users don't even have to have a compiler.

scikit-image is on a "must have cython to install from source" model as well.

Reply:
Wheels sound like a good idea, but may not be so easy.

Reply:
I think I know how to do the linux ones, it looks like CGohlke is already building the windows ones (we just need to upload), the mac ones are the only problem.  Would prefer to find a mac-developer to build them.

I disagree that this is a breaking change, packaging / installing is not part of the API.

Reply:
You have a point. I am OK with this PR then.